README.md

```diff
@@ -1,3 +1,50 @@
 # DeployQA
 
-DeployQA is a tool for quick deployment of QA environments.
+DeployQA is an automated tool for application deployment. It helps teams to make deployment and testing processes simple and easy to maintain.
+
+## Installation
+ 
+ - Create Github Application
+ - Create Slack Application
+ - Create Heroku account
+ 
+#### Configure Github App
```

Comment:
There is also a permissions:
 - content - read
 - issues - read/write
 - pull requests - read/write

and webhooks:
 - pull requests

Reply:
exactly 👍